### ISSUE TITLE
Include new prettify

### ISSUE THREAD

**Issue by alazzaro** [cp2k/dbcsr#95]
New prettify from https://github.com/pseewald/fprettify as submodule
Unfortunately, the new prettify "forces" make to always run the target since it always updates the file, even there were no changes!

**Comment by alazzaro** [comment]
Close with [972f09e](https://github.com/cp2k/dbcsr/commit/972f09eb17c021c4970ddbd08596869b43e88a33)

**Comment by alazzaro** [comment]
Sure, of course. My first step was to include it...

**Comment by alazzaro** [comment]
@pseewald @dev-zero What's the status of this issue?
My trick was to do this in the Makefile:

```
# Pretty function, check if the file requires update
define pretty_func
        @mkdir -p $(PRETTYOBJDIR)
        @rm -f $2
        $(TOOLSRC)/fprettify/fprettify.py --disable-whitespace -s $1 > $2
        @cmp -s $1 $2; \
        RETVAL=$$?; \
        if [ $$RETVAL -ne 0 ]; then \
            cp $2 $1; \
        fi
endef
```

but I think we can now remove it with the new prettifier, right?

**Comment by pseewald** [comment]
I still need to fix pseewald/fprettify#34

**Comment by alazzaro** [comment]
OK, then I will move this issue to v3. No rush.

**Comment by pseewald** [comment]
> I still need to fix [pseewald/fprettify#34](https://github.com/pseewald/fprettify/issues/34)

This is fixed

**Comment by alazzaro** [comment]
closed via #270
